build/media-suite_helper.sh

```diff
@@ -188,7 +188,10 @@ do_vcs() {
                 ref=${vcsBranch##*=}
                 ;;
             branch)
-                ref=origin/${vcsBranch##*=}
+                if [[ $vcsType = hg ]]; then
+                    ref=${vcsBranch##*=}
+                else
+                    ref=origin/${vcsBranch##*=}
```

Comment:
I'd prefer something like
```diff
diff --git a/build/media-suite_helper.sh b/build/media-suite_helper.sh
index f661f4e..468e36c 100644
--- a/build/media-suite_helper.sh
+++ b/build/media-suite_helper.sh
@@ -188,7 +188,8 @@ do_vcs() {
                 ref=${vcsBranch##*=}
                 ;;
             branch)
-                ref=origin/${vcsBranch##*=}
+                ref=${vcsBranch##*=}
+                [[ $vcsType = git && $ref = ${ref%/*} ]] && ref=origin/$ref
                 ;;
         esac
     else
```

Reply:
That works for me just fine as well so I've change it to your code, thank you! :+1: